bin/uniref90_rhea_filter.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import gzip
+
+from Bio import SeqIO
+
+
+def load_mapping(tsv_file):
+    """
+    Load mapping of UniRef90 proteins to Rhea IDs from a TSV file.
+    Returns a dictionary with RepID as the key and RheaID as the value.
+    """
+    mapping = {}
+    with open(tsv_file, 'r') as file:
+        for line in file:
+            rep_id, rhea_id = line.strip().split('\t')
+            rhea_id = " ".join(sorted(set(rhea_id.split())))
+            mapping[rep_id] = rhea_id
+    return mapping
```

Comment:
Please use the csv module to parse csv/tsv files

Reply:
Look at the DictReader, it may be handly in this case

Reply:
Fixed the function in [use csv module to parse file](https://github.com/EBI-Metagenomics/taxdb_generation_nf/pull/1/commits/5637896c180500e883a44197496ca7dd826ff924). I tried DictReader, but default reader seems more readable to me in this case